rootfs-version-check/module/rootfs-version-check

```diff
@@ -22,6 +22,17 @@ fi
 
 case "$1" in
     Download)
+        if [ ! -x /usr/bin/mender-compare-versions ]; then
```

Comment:
Should this be put in e.g `/usr/share/mender/version/mender-compare-versions` to align where other Mender executable are?

Reply:
Fine by me. I'll make that modification and update both PRs.